examples/x86_64.swiftinterface

```diff
@@ -0,0 +1,507 @@
+  public let dispatchQueue: Dispatch.DispatchQueue?
+  public enum ApiVersion : AzureCore.RequestStringConvertible {
+    case custom(Swift.String)
+    case v20210307
+    public static var latest: AzureCommunicationChat.AzureCommunicationChatClientOptions.ApiVersion {
+      get
+    }
+    public var requestString: Swift.String {
+      get
+    }
+    public init(_ val: Swift.String)
+  }
+  public init(apiVersion: AzureCommunicationChat.AzureCommunicationChatClientOptions.ApiVersion = .latest, logger: AzureCore.ClientLogger = ClientLoggers.default(tag: "AzureCommunicationChat"), telemetryOptions: AzureCore.TelemetryOptions = TelemetryOptions(), transportOptions: AzureCore.TransportOptions? = nil, dispatchQueue: Dispatch.DispatchQueue? = nil)
+}
+public struct ChatThreadItem : Swift.Codable {
+  public let id: Swift.String
+  public let topic: Swift.String
+  public let deletedOn: AzureCore.Iso8601Date?
+  public let lastMessageReceivedOn: AzureCore.Iso8601Date?
+  public init(id: Swift.String, topic: Swift.String, deletedOn: AzureCore.Iso8601Date? = nil, lastMessageReceivedOn: AzureCore.Iso8601Date? = nil)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+public struct PhoneNumberIdentifierModel : Swift.Codable {
+  public let value: Swift.String
+  public init(value: Swift.String)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+public struct AddChatParticipantsResult : Swift.Codable {
+  public let invalidParticipants: [AzureCommunicationChat.ChatError]?
+  public init(invalidParticipants: [AzureCommunicationChat.ChatError]? = nil)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+public typealias TrouterEventHandler = (AzureCommunicationChat.TrouterEvent) -> Swift.Void
+public enum TrouterEvent {
+  case chatMessageReceivedEvent(AzureCommunicationChat.ChatMessageReceivedEvent)
+  case typingIndicatorReceived(AzureCommunicationChat.TypingIndicatorReceivedEvent)
+  case readReceiptReceived(AzureCommunicationChat.ReadReceiptReceivedEvent)
+  case chatMessageEdited(AzureCommunicationChat.ChatMessageEditedEvent)
+  case chatMessageDeleted(AzureCommunicationChat.ChatMessageDeletedEvent)
+  case chatThreadCreated(AzureCommunicationChat.ChatThreadCreatedEvent)
+  case chatThreadPropertiesUpdated(AzureCommunicationChat.ChatThreadPropertiesUpdatedEvent)
+  case chatThreadDeleted(AzureCommunicationChat.ChatThreadDeletedEvent)
+  case participantsAdded(AzureCommunicationChat.ParticipantsAddedEvent)
+  case participantsRemoved(AzureCommunicationChat.ParticipantsRemovedEvent)
+}
+public struct UpdateChatMessageRequest : Swift.Codable {
+  public let content: Swift.String?
+  public init(content: Swift.String? = nil)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
```

Comment:
Also no references in the public API. Should be internal.

Reply:
This will be public in preview6 btw (related to your other question) - I'll regenerate with updated autorest config & make this internal